package.json

```diff
@@ -26,117 +26,114 @@
-    "@emotion/react": "~11.10.6",
-    "@emotion/styled": "~11.10.6",
-    "@graasp/apps-query-client": "2.0.0-rc.1",
-    "@graasp/sdk": "1.0.0-rc1",
-    "@graasp/ui": "3.0.0-rc.2",
-    "@mui/icons-material": "~5.11.11",
-    "@mui/lab": "~5.0.0-alpha.124",
-    "@mui/material": "~5.11.14",
-    "@sentry/react": "~7.44.2",
-    "@sentry/tracing": "~7.44.2",
-    "@tanstack/react-query": "4.28.0",
-    "@tanstack/react-query-devtools": "4.28.0",
-    "@uppy/core": "~3.1.1",
-    "@uppy/dashboard": "~3.3.1",
-    "@uppy/drag-drop": "~3.0.1",
-    "@uppy/file-input": "~3.0.1",
-    "@uppy/progress-bar": "3.0.1",
-    "@uppy/react": "~3.1.0",
-    "@uppy/xhr-upload": "~3.1.0",
-    "dompurify": "3.0.1",
+    "@emotion/react": "~11.11.1",
+    "@emotion/styled": "~11.11.0",
+    "@graasp/apps-query-client": "2.0.1",
+    "@graasp/sdk": "1.1.3",
+    "@graasp/ui": "3.2.7",
+    "@mui/icons-material": "~5.14.1",
+    "@mui/lab": "~5.0.0-alpha.137",
+    "@mui/material": "~5.14.2",
+    "@sentry/react": "~7.60.1",
+    "@sentry/tracing": "~7.60.1",
+    "@tanstack/react-query": "4.32.0",
+    "@tanstack/react-query-devtools": "4.32.0",
+    "@uppy/core": "~3.3.1",
+    "@uppy/dashboard": "~3.5.0",
+    "@uppy/drag-drop": "~3.0.2",
+    "@uppy/file-input": "~3.0.2",
+    "@uppy/progress-bar": "3.0.2",
+    "@uppy/react": "~3.1.3",
+    "@uppy/xhr-upload": "~3.3.1",
+    "dompurify": "3.0.5",
     "file-saver": "~2.0.5",
-    "i18next": "~22.4.13",
+    "i18next": "~22.4.15",
     "immutable": "4.3.0",
     "lodash.isobject": "~3.0.2",
     "lodash.isstring": "~4.0.1",
     "react": "~18.2.0",
     "react-dom": "~18.2.0",
     "react-draggable": "4.4.5",
-    "react-i18next": "~12.2.0",
+    "react-i18next": "~12.2.2",
     "react-quill": "2.0.0",
-    "react-router-dom": "~6.9.0",
-    "react-scripts": "~5.0.1",
-    "react-toastify": "~9.1.2",
+    "react-router-dom": "~6.14.2",
+    "react-toastify": "~9.1.3",
     "react-transition-group": "4.4.5",
     "react-zoom-pan-pinch": "~2.6.1",
     "typescript": "~4.8.4",
     "uuid": "~9.0.0"
   },
   "scripts": {
-    "start": "env-cmd -f ./.env.local react-scripts --max_old_space_size=4096 -r @cypress/instrument-cra start",
-    "start:dev": "env-cmd -f ./.env.development react-scripts --max_old_space_size=4096 -r @cypress/instrument-cra start",
-    "start:test": "env-cmd -f ./.env.test react-scripts --max_old_space_size=4096 -r @cypress/instrument-cra start",
-    "start:ci": "react-scripts --max_old_space_size=4096 -r @cypress/instrument-cra start",
-    "build": "react-scripts --max_old_space_size=4096 build",
-    "build:dev": "env-cmd -f ./.env.dev react-scripts --max_old_space_size=4096 build",
-    "test:cra": "react-scripts test",
-    "eject": "react-scripts eject",
+    "dev:mock": "VITE_MOCK_API=true && yarn vite",
+    "dev": "yarn vite",
+    "start:test": "yarn vite --mode test",
+    "build": "NODE_OPTIONS=--max_old_space_size=16384 && yarn vite build",
```

Comment:
You should not need that option. Try removing it, and checking your versions of sdk, immutable etc to see if they match.